.github/workflows/deb-daily-builds.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt update -qq
-          sudo apt install -qq -y python3-launchpadlib
+          sudo apt install -qq -y python3-launchpadlib python3-setuptools-scm
           git clone -b main https://git.launchpad.net/~hook25/ppa-dev-tools /tmp/ppa-dev-tools
```

Comment:
Is this fork of `ppa-dev-tools` going to be long lived?

Reply:
Unrelated to this PR, but we could actually go back to the upstream version now that every change I made is in main